spec/support/test_env.rb

```diff
@@ -81,7 +80,7 @@ def setup_factory_repo
     end
 
     # We must copy bare repositories because we will push to them.
-    system(*%W(git clone --bare #{factory_repo_path} #{factory_repo_path_bare}))
+    system(*%W(git clone -q --bare #{factory_repo_path} #{factory_repo_path_bare}))
```

Comment:
Didn't bother with this one. You could change the `system` call to backticks and remove the `%W()` declaration, but I thought I remembered a GitLab coding guideline about always using this syntax for external processes.